### PR TITLE
Couple of header include fixes

### PR DESCRIPTION
**Pull request by rpurdie**
This series fixes a couple of minor missing include files.

### CODE REVIEW

File: src/login/logind-brightness.c [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
+#include <sys/wait.h>
```
Comment (poettering) [inline src/login/logind-brightness.c:3]: process-util already pulls that in?

File: src/basic/copy.c [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 #include <errno.h>
 #include <fcntl.h>
+#include <signal.h>
```
Comment (poettering) [inline src/basic/copy.c:5]: missing_syscall already pulls that in?

**Review by poettering — CHANGES_REQUESTED** [review]
both cases are redundant? how did you run into this?

**Comment by rpurdie** [comment]
Sorry, these were needed in an older version but are fixed with current master, this is invalid but I can clean some things up, sorry for the noise.

**Pull request closed without merging**